src/main/kotlin/com/atlassian/performance/tools/report/chart/waterfall/Utils.kt

```diff
@@ -28,4 +29,12 @@ internal class Utils {
             label
         }
     }
-}
\ No newline at end of file
+
+    fun toValidUri(urlStr: String): URI {
+        return try {
+            URI(urlStr)
+        } catch (e: URISyntaxException) {
+            URI(urlStr.substringBefore(','))
```

Comment:
https://en.wikipedia.org/wiki/Uniform_Resource_Identifier
> A Uniform Resource Identifier (URI) is a string of characters that unambiguously identifies a particular resource.

`data:image/svg+xml;base64` does not identify a resource. It's just a piece of an URI.
https://en.wikipedia.org/wiki/Data_URI_scheme
The `<data>` is mandatory.
We don't even need to promote this `String` to an `URI`, because we're not using it to identify anything.

Reply:
PS. It's a shame that the `URI` constructor does not accept a valid URI.